documentation/modules/con-securing-kafka-authorization.adoc

```diff
@@ -30,12 +33,13 @@ authorization:
 # ...
 ----
 
-Access rules for users are xref:simple-acl-str[defined using Access Control Lists (ACLs)].
-You can optionally designate a list of super users in the `superUsers` field.
+Access rules for users are xref:type-AclRule-reference[defined using Access Control Lists (ACLs)].
 
 [id='ref-kafka-authorization-super-user-{context}']
 == Super users
```

Comment:
Should we make it clear that this is supported with all 3 authorization mechanisms?

Reply:
Yes. I've updated.

Reply:
I've also updated the links from the appendix from simple and opa schema refs